### PR TITLE
[COST-4715] sentry -> glitchtip cleanup p1

### PR DESCRIPTION
**Pull request by maskarb**
## Jira Ticket

[COST-4715](https://issues.redhat.com/browse/COST-4715)

## Description
* replace `API_SENTRY_DSN_KEY`, `CELERY_SENTRY_DSN_KEY`, `LISTENER_SENTRY_DSN_KEY`, `SOURCE_SENTRY_DSN_KEY` with `GLITCHTIP_KEY_NAME`
* Add new variables (`KOKU_ENABLE_SENTRY`, `KOKU_SENTRY_DSN`, and `KOKU_SENTRY_ENVIRONMENT`) which will consolidate several different SENTRY env variables.

This PR is written so that the old variables are still used. Once we switch stage/prod over to the new variables, we can remove them entirely.

## Release Notes
- [x] proposed release note

```markdown
* [[COST-4715](https://issues.redhat.com/browse/COST-4715)] Create new environment variables to consolidate the Glitchtip configuration
```

### CODE REVIEW

File: koku/koku/sentry.py [hunk 1]
```diff
@@ -25,7 +25,15 @@ def traces_sampler(sampling_context):
...
 
-if ENVIRONMENT.bool("KOKU_API_ENABLE_SENTRY", default=False):
+if ENVIRONMENT.bool("KOKU_ENABLE_SENTRY", default=False):
+    sentry_sdk.init(
+        dsn=ENVIRONMENT("KOKU_SENTRY_DSN"),
+        environment=ENVIRONMENT("KOKU_SENTRY_ENVIRONMENT"),
+        traces_sampler=traces_sampler,
+    )
+    LOG.info("Sentry setup.")
+# TODO: remove the following in favor of the above `if`
```
Comment (djnakabaale) [inline koku/koku/sentry.py:35]: Do you plan to remove this `TODO` in a followup pr?

File: koku/koku/sentry.py [hunk 2]
```diff
@@ -34,6 +42,7 @@ def traces_sampler(sampling_context):
         traces_sampler=traces_sampler,
     )
     LOG.info("Sentry setup.")
+# TODO: remove the following in favor of the above `if`
```
Comment (djnakabaale) [inline koku/koku/sentry.py:45]: Do you plan to remove this `TODO` in a followup pr?

**Review by djnakabaale — COMMENTED** [review]
This looks good to me. Here are some nitpick question:

**Review by djnakabaale — APPROVED** [review]
ltgm! 👍🏾

File: koku/koku/sentry.py (hunk 1 above)
Comment (maskarb) [reply koku/koku/sentry.py:35]: Yes, these cannot be removed until the other envs are switched to use the new variables.

**Pull request merged**